scilpy/reconst/raw_signal.py

```diff
@@ -61,7 +61,11 @@ def compute_sh_coefficients(dwi, gradient_table, sh_order=8,
     # Ensure that this is on a single shell.
     shell_values, _ = identify_shells(bvals)
     shell_values.sort()
-    if shell_values.shape[0] != 2 or shell_values[0] > 20.:
+    if force_b0_threshold:
```

Comment:
It's actually the opposite (I know it's counter-intuitive...): if `force_b0_threshold` is given, we should use `bvals.min()` as the threshold (see function `check_b0_threshold`: https://github.com/scilus/scilpy/blob/master/scilpy/utils/bvec_bval_tools.py#L61)